pkg/render/logstorage_test.go

```diff
@@ -267,6 +267,99 @@ var _ = Describe("Elasticsearch rendering tests", func() {
 
 		Context("Deleting LogStorage", deleteLogStorageTests(operatorv1.ClusterManagementTypeStandalone))
 
+		Context("Updating LogStorage resource", func() {
```

Comment:
Does this test fail without your fix? I'm not sure if the fake k8s client we use actually captures the problem.

Reply:
Could you add an assertion on a spec change?

Reply:
Everything about this fix hinges on the function `nodeSetName()` generating a unique (20-char) hash substring given corev1.PersistentVolumeClaim instances that contain different values. 

Would like to see a few more tests perhaps just for `nodeSetName()` itself (unit style tests) to illustrate this. Are there an edge cases where this isn't true?